storybookjs/storybook
# Controls: Style refinements

## What I did
Minor style refinements to controls date, boolean, and input. 

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes. Look at UI Review PR check.
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No
